src/less/actionToolbar.less

```diff
@@ -3,7 +3,7 @@
 // --------------------------------------------------
 
 .action-area {
-  background-color: @action-toolbar-bg;
+  background: @gray-75;
```

Comment:
any reason we're using @gray-75 as the value here instead of defining @gray-75 as the value of @action-toolbar-bg?